core/src/test/java/apoc/util/UtilTest.java

```diff
@@ -79,14 +80,12 @@ public void mergeNullMaps() {
         assertNotNull(Util.merge(null, null));
     }
 
-    @Test(expected = QueryExecutionException.class)
+    @Test
     public void testValidateQuery() {
-        try {
-            Util.validateQuery(db, "Match (n) return m");
-        } catch (QueryExecutionException e) {
-            assertTrue(e.getMessage().contains("Variable `m` not defined"));
-            throw e;
-        }
+        QueryExecutionException e = assertThrows(QueryExecutionException.class,
+                () -> Util.validateQuery(db, "Match (n) return m")
```

Comment:
Not really a part of the PR, but it kinda bothers me our Cypher is written this way, would be nice to have it match the style guide, but super low prio I guess 🤷‍♀️

Reply:
Fixed this one, if you want to do it for more places you can create a DayOFCleanup card